### PR TITLE
PLG-154: DSM - Update the Dropdown page

### PR DESCRIPTION
**Pull request by BryanS29-akeneo**
**Description (for Contributor and Core Developer)**

Add "No results" information in the Dropdown component.

**Definition Of Done (for Core Developer only)**

- [ ] Tests
- [ ] Migration & Installer
- [ ] PM Validation (Story)
- [ ] Changelog (maintenance bug fixes)
- [ ] Tech Doc

### CODE REVIEW

File: front-packages/akeneo-design-system/src/components/Dropdown/Dropdown.stories.mdx [hunk 1]
```diff
@@ -244,7 +271,7 @@ For accessibility reasons, we must put the full text in an HTML title attribute.
                     title="Search"
                   />
                 </Dropdown.Header>
-                <Dropdown.ItemCollection onNextPage={handleNextPage}>
+                <Dropdown.ItemCollection onNextPage={handleNextPage} noResultLabel="Sorry, there is no results.">
```
Comment (antoinetrouve) [inline front-packages/akeneo-design-system/src/components/Dropdown/Dropdown.stories.mdx:274]: The noResultLabel value is free to be override ? Is this wanted ?

Comment (BryanS29-akeneo) [reply front-packages/akeneo-design-system/src/components/Dropdown/Dropdown.stories.mdx:274]: Yes this is wanted, we have to pass the label in a props.

File: front-packages/akeneo-design-system/src/components/Dropdown/ItemCollection/ItemCollection.tsx [hunk 2]
```diff
@@ -62,7 +80,14 @@ const ItemCollection = React.forwardRef<HTMLDivElement, ItemCollectionProps>(
 
     return (
       <ItemCollectionContainer {...rest} ref={containerRef}>
-        {decoratedChildren}
+        {childrenCount
+          ? decoratedChildren
+          : noResultLabel && (
+              <NoResultSection>
+                <GroupsIllustration size={128} />
```
Comment (pierallard) [inline front-packages/akeneo-design-system/src/components/Dropdown/ItemCollection/ItemCollection.tsx:87]: It's really weird to force this illustration
For example, we don't have this one in our code
https://github.com/akeneo/pim-enterprise-dev/blob/master/grth/src/Akeneo/Pim/TableAttribute/front/src/attribute/ManageOptionsModal.tsx#L425

Comment (BryanS29-akeneo) [reply front-packages/akeneo-design-system/src/components/Dropdown/ItemCollection/ItemCollection.tsx:87]: I follow this mockup : https://app.abstract.com/projects/25bb2690-25f8-11e9-aae0-5d753cf86983/branches/master/commits/b3a6882636ead199ab77bff5f82cb38c9f6fff30/files/1DAEA0CF-7551-443E-9273-71C14413762E/layers/599AB732-35CE-439F-94D6-C55BA8CB5F4F

Do I have to use the Placeholder component and pass the GroupsIllustration in it ?

Comment (pierallard) [reply front-packages/akeneo-design-system/src/components/Dropdown/ItemCollection/ItemCollection.tsx:87]: I can't access to your mockup, but @yulldesign ask us to create this "Placeholder" for any place where there are nothing, including Dropdown, and we currently use them in our bundle.
What disturbs me is "is it always this design when there are no results" ?
If yes, this has to be in the DSM ; if no, I think this should not. @yulldesign should take this decision

Comment (BryanS29-akeneo) [reply front-packages/akeneo-design-system/src/components/Dropdown/ItemCollection/ItemCollection.tsx]: As see with @yulldesign, I updated with the Placeholder component. Can you have a look @pierallard please ?

**Review by pierallard — APPROVED** [review]

**Review by LaurentPetard — APPROVED** [review]

**Pull request merged**